### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by birkoffe**

### CODE REVIEW

File: lang/1049.json [hunk 1]
```diff
@@ -194,6 +194,10 @@
         "regex": "Регулярное выражения",
         "apply": "Применить",
         "maximized": "На весь экран",
-        "no_magnet_links_found": "Magnet-ссылки не найдены"
+        "no_magnet_links_found": "Magnet-ссылки не найдены",
+        "notification_area": "Область уведомлений",
+        "show_picotorrent_in_notification_area": "Показывать PicoTorrent в области уведомлений",
+        "minimize_to_notification_area": "Сворачивать в область уведомлений",
+        "close_to_notification_area": "Закзывать в область уведомлений"
```
Comment (DoumanAsh) [inline lang/1049.json]: >Закзывать

Закрывать

**Comment by vktr** [comment]
@birkoffe @DoumanAsh Will you let me know when this is ready to merge? 😄

**Comment by DoumanAsh** [comment]
I don't have any more comments

**Comment by birkoffe** [comment]
Yes, you can merge.

**Pull request merged**